optimum/intel/openvino/modeling_decoder.py

```diff
@@ -227,6 +227,34 @@ def _from_transformers(
             if use_cache:
                 task = task + "-with-past"
 
+        # Patch the modules to export of GPTQ models w/o GPU
+        do_gptq_patching = False
+        config_dict = config.to_dict()
+        quantization_config = config_dict.get("quantization_config", None)
+        do_gptq_patching = quantization_config and quantization_config["quant_method"] == "gptq"
+        if do_gptq_patching:
+            torch.set_default_dtype(torch.float32)
+            orig_cuda_check = torch.cuda.is_available
+            torch.cuda.is_available = lambda: True
```

Comment:
why is this needed ?

Reply:
GPTQ-optimized models from Transformers require NVidia GPU to be available for inference. We are introducing two changes to overcome this limitation:
- Patching some functions that prevent instantiating AutoModelForCausalLM without GPUs
- Patching QLinear forward in AutoGPTQ to enable model tracing via TorchScript

The first part comes to Optimum while the second is to OpenVINO PyTorch Frontend.

Reply:
thanks for the explanation !